### PR TITLE
Added unittest for GraphML

### PR DESCRIPTION
**Pull request by vtraag**
This PR adds a unit test to test reading GraphML. The test is skipped if the functionality is not available from the C core.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -259,6 +259,7 @@ def compile_in(self, source_folder, build_folder, install_folder):
         # Finally, add the source folder path
         args.append(str(build_to_source_folder))
 
+        print("Running {}".format(" ".join(args)))
```
Comment (ntamas) [inline setup.py:262]: Can we live without this line or did you need this line often during debugging? (I can leave it in if it was needed so often that you'd rather have it in the script).

File: tests/test_foreign.py [hunk 2]
```diff
@@ -339,9 +399,6 @@ def testPickle(self):
             self.assertTrue(isinstance(g, Graph))
             self.assertTrue(g.vcount() == 3 and g.ecount() == 1 and not g.is_directed())
             g.write_pickle(tmpfname)
-
-    
-
```
Comment (ntamas) [inline tests/test_foreign.py:344]: Some separation would be great between test cases. I agree that three lines is maybe too much, but no newlines at all may be too little :)

File: setup.py (hunk 1 above)
Comment (vtraag) [reply setup.py:262]: No, we can remove it. I just added it for debugging, but I didn't thought it hurt to be explicit about it.

File: tests/test_foreign.py (hunk 2 above)
Comment (vtraag) [reply tests/test_foreign.py:344]: Ha, oh sorry, must have removed those lines accidentally :)

**Review by ntamas — APPROVED** [review]

**Comment by ntamas** [comment]
Thanks!

**Pull request merged**